src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -233,10 +238,202 @@
+    <Compile Include="mono\System.Drawing\GraphicsUnit.cs" />
+    <Compile Include="mono\System.Drawing\gdipEnums.cs" />
+    <Compile Include="mono\System.Drawing\gdipFunctions.cs" />
+    <Compile Include="System\Drawing\GdiplusNative.Unix.cs" />
+    <Compile Include="mono\System.Drawing\gdipStructs.cs" />
+    <Compile Include="mono\System.Drawing\Icon.cs" />
+    <Compile Include="mono\System.Drawing\ImageAnimator.cs" />
+    <Compile Include="mono\System.Drawing\Image.cs" />
+    <Compile Include="mono\System.Drawing\KnownColors.cs" />
+    <Compile Include="mono\System.Drawing\Pen.cs" />
+    <Compile Include="mono\System.Drawing\Pens.cs" />
+    <Compile Include="mono\System.Drawing\Region.cs" />
+    <Compile Include="mono\System.Drawing\SRDescriptionAttribute.cs" />
+    <Compile Include="mono\System.Drawing\StringFormat.cs" />
+    <Compile Include="mono\System.Drawing\SystemColors.cs" />
+    <Compile Include="mono\System.Drawing\SystemFonts.cs" />
+    <Compile Include="mono\System.Drawing\SystemIcons.cs" />
+    <Compile Include="mono\System.Drawing\SystemPens.cs" />
+    <Compile Include="mono\System.Drawing\ToolboxBitmapAttribute.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\GraphicsContainer.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\GraphicsPath.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\LinearGradientBrush.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\Matrix.cs" />
+    <Compile Include="mono\System.Drawing.Drawing2D\PathGradientBrush.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\BitmapData.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\ImageFormat.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\Metafile.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\MetafileHeader.cs" />
+    <Compile Include="mono\System.Drawing.Imaging\MetaHeader.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintingServices.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintingServicesWin32.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintingServicesUnix.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PageSettings.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PreviewPrintController.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintController.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintDocument.cs">
+      <SubType>Component</SubType>
+    </Compile>
+    <Compile Include="mono\System.Drawing.Printing\PrinterSettings.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintEventArgs.cs" />
+    <Compile Include="mono\System.Drawing.Printing\PrintPageEventArgs.cs" />
+    <Compile Include="mono\System.Drawing.Printing\StandardPrintController.cs" />
+    <Compile Include="mono\System.Drawing.Text\PrivateFontCollection.cs" />
+    <Compile Include="mono\MonoTODOAttribute.cs" />
+    <!-- Shared with Windows -->
```

Comment:
How about splitting the files into ItemGroups based on whether the files are Windows-specific, Unix-specific, or platform-agnostic?